catalog/video/catalog-mpegts-transformer/src/main/java/com/connexta/transformer/video/LocationKlvProcessor.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+import java.util.Map;
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.io.WKTReader;
+import com.vividsolutions.jts.io.WKTWriter;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.impl.AttributeImpl;
+
+/**
+ * Generate the location metadata based on the klv corner data. Callers must supply a
+ * {@link com.connexta.transformer.video.KlvProcessor.Configuration} that contains a postive (>0)
+ * Integer for {@link com.connexta.transformer.video.KlvProcessor.Configuration#SUBSAMPLE_COUNT}.
+ */
+public class LocationKlvProcessor implements KlvProcessor {
+
+    public static final Integer MIN_SUBSAMPLE_COUNT = 1;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(LocationKlvProcessor.class);
+
+    private Optional<KlvHandler> find(Map<String, KlvHandler> handlers, String name) {
+        return handlers.values()
+                .stream()
+                .filter(handler -> handler.getAttributeName()
+                        .equals(name))
+                .findFirst();
+    }
+
+    @Override
+    public void process(Map<String, KlvHandler> handlers, Metacard metacard,
+            Configuration configuration) {
+
+        if (configuration.get(Configuration.SUBSAMPLE_COUNT) == null || (Integer) configuration.get(
+                Configuration.SUBSAMPLE_COUNT) < MIN_SUBSAMPLE_COUNT) {
+            LOGGER.error(
```

Comment:
Is there any project documentation that describes which situations should be error vs warn?